sherpa/tests/test_data.py

```diff
@@ -707,19 +707,17 @@ def test_data2_get_x0(data):
 
 @pytest.mark.parametrize("data", (Data2DInt, ), indirect=True)
 def test_data2_int_get_x0(data):
-    actual = data.get_x1()
-    numpy.testing.assert_array_equal(actual, X1_2D)
+    numpy.testing.assert_array_equal(data.get_x0(), X0_2D)
 
 
 @pytest.mark.parametrize("data", (Data2D, ), indirect=True)
 def test_data2_get_x1(data):
-    numpy.testing.assert_array_equal(data.get_x0(), X0_2D)
+    numpy.testing.assert_array_equal(data.get_x1(), X1_2D)
 
 
 @pytest.mark.parametrize("data", (Data2DInt, ), indirect=True)
 def test_data2_int_get_x1(data):
-    actual = data.get_x1()
-    numpy.testing.assert_array_equal(actual, X1_2D)
+    numpy.testing.assert_array_equal(data.get_x1(), X1_2D)
```

Comment:
I note that these two tests can now we summarized as:
```
@pytest.mark.parametrize("data", Data2DClasses, indirect=True)
def test_data2_get_x1(data):
    numpy.testing.assert_array_equal(data.get_x1(), X1_2D)
```
That's not important (it only saves three lines) so I'm approving, but if @DougBurke wants to adjust that before merging thats also fine.

Reply:
There's a number of other tests that would benefit from this cleanup, but I don't have the time/energy to do this so I'd rather not do it here.